test/tests/userobjects/pre_aux_based_on_exec_flag/pre_post_aux_test.i

```diff
@@ -60,7 +60,7 @@
     source_variable = u
     shift = -100.0
     normalization = 1.0
-    execute_on = 'INITIAL TIMESTEP_BEGIN TIMESTEP_END FINAL'
+    execute_on = 'INITIAL FINAL'
```

Comment:
why these changes?

Reply:
Possibly unrelated trivial change. This aux kernel only depends on a primal variable thus does not have dependency issue. I possibly just made this change when trying to output on final.